packages/manager/apps/telecom/src/components/telecom/telephony/group/consumption/pie-chart/telephony-group-consumption-pie-chart.html

```diff
@@ -1,4 +1,4 @@
-<div class="pie__legend">
+<div class="{{pieLegendClass()}}">
```

Comment:
maybe you can use ng-class

Reply:
I've try with ng-class but i've fail. What i put in ng-class isn't interpreted and i have the function name on class name place :/
i have try : 
 - data-ng-class="pieLegendClass()"
 - data-ng-class="toggle ? 'pie__legend' : 'pie__legend__subtitle'"

this 2 isn't interpreted and came on DOM.

Reply:
@Heiiji what about that:
pie__legend__subtitle' : 'pie__legend
```suggestion
<div class="{ 'pie__legend': toggle, 'pie__legend__subtitle': !toggle }">
```

Reply:
yeah, thanks :) i've push this modif and it's work. I'm not very familliar with angular DOM logic for moment. ^^